lib/gollum-lib/macro/video.rb

```diff
@@ -0,0 +1,17 @@
+module Gollum
+  class Macro
+    class Video < Gollum::Macro
+      def render (fname)
+        begin 
```

Comment:
Actually, according to [this](http://stackoverflow.com/a/7537309) the `MIME::Types.type_for` call only determines mime type based on file extension, not on content anyway. So I would suggest we just say that's good enough: if you want to use video's in your wiki, you should use the appropriate file extensions. That allows for the simplest implementation. Perhaps we can still have an optional `mime` argument for the Macro in case the user wants to override the extension-detected type.